ballerina-tests/build.gradle

```diff
@@ -85,6 +86,15 @@ task initializeVariables {
     if (project.hasProperty("balJavaDebug")) {
         balJavaDebugParam = "BAL_JAVA_DEBUG=${project.findProperty("balJavaDebug")}"
     }
+    if (project.hasProperty('balNativeTest')) {
+        nativeFlag = '--native'
+    }
+
+    gradle.taskGraph.whenReady { graph ->
+        if (!graph.hasTask(":${packageName}-${packageOrg}:test")) {
+            testParams = "--skip-tests"
```

Comment:
What is the purpose of this flag?

Reply:
Removed with [688bfbf738690fd979123fd14145ee0793d089f8](https://github.com/ballerina-platform/module-ballerina-websub/pull/572/commits/688bfbf738690fd979123fd14145ee0793d089f8)